packages/base-conf/fstab

```diff
@@ -1,2 +1,2 @@
-LABEL=COS_PERSISTENT    /usr/local  auto    nofail,x-systemd.device-timeout=1ms,x-systemd.requires-mounts-for=/etc,x-systemd.wanted-by=local-fs.target    0  2
-LABEL=COS_OEM           /oem        auto    nofail,x-systemd.device-timeout=1ms,x-systemd.requires-mounts-for=/etc,x-systemd.wanted-by=local-fs.target    0  2
+LABEL=COS_PERSISTENT    /usr/local  auto    nofail,x-systemd.device-timeout=5s,x-systemd.wanted-by=local-fs.target    0  2
```

Comment:
Just wondering if we will need the same for `COS_PERSISTENT`? As we let the user store `cloud-config` files also in `/usr/local/cloud-config`

Reply:
Oh I did not consider this scenario, but yes, if we let the user place init files in `/usr/local/cloud-config` we should do the same there.

Reply:
let me adapt it.

Reply:
done